File: 1. TESTE DE WEB SCRAPING/scraping_java/src/main/java/com/scraping/java/directory/DirectoryManager.java
package com.scraping.java.directory;

import com.scraping.java.LogsMain;

import java.io.File;

public class DirectoryManager {
    public static void createDirectory(String path) {
        File dir = new File(path);
        if (!dir.exists()) {
            dir.mkdirs();
            LogsMain.logInfo("Diretório criado: " + path);
        } else {
            LogsMain.logWarning("Diretório já existe: " + path);
        }
    }
}

File: 1. TESTE DE WEB SCRAPING/scraping_python/src/directory/directory_manager.py
import os

class DirectoryManager:
    @staticmethod
    def create_directory(path):
        green = "\033[92m"
        reset = "\033[0m"

        if not os.path.exists(path):
            os.makedirs(path)
            print(f"{green}Diretório criado: {path}{reset}")
        else:
            print(f"{green}Diretório já existe: {path}{reset}")
